src/jit/gentree.cpp

```diff
@@ -747,18 +747,13 @@ int GenTree::GetRegisterDstCount() const
     {
         return (const_cast<GenTree*>(this))->AsPutArgSplit()->gtNumRegs;
     }
-    // A PUTARG_REG could be a MultiRegOp on ARM since we could move a double register to two int registers,
-    // either for all double parameters w/SoftFP or for varargs).
-    else
+#endif
+#if !defined(_TARGET_64BIT_)
+    else if (OperIsMultiRegOp())
```

Comment:
Seems like this deserves a comment?

It is confusing that a multi reg op might only have one register destination.

Reply:
Modified the code a bit and added a comment; on x86 it is always 2 (just MUL_LONG).